applications/matter_weather_station/src/app_task.cpp

```diff
@@ -99,20 +103,24 @@ int AppTask::Init()
 
 	/* Initialize timers */
 	k_timer_init(
-		&sFunctionTimer, [](k_timer *) { sAppTask.PostEvent(AppEvent::Type::kTimer, FunctionTimerHandler); },
+		&sFunctionTimer,
+		[](k_timer *) { sAppTask.PostEvent(AppEvent{ AppEvent::FunctionTimer }); },
 		nullptr);
 	k_timer_init(
 		&sMeasurementsTimer,
-		[](k_timer *) { sAppTask.PostEvent(AppEvent::Type::kTimer, MeasurementsTimerHandler); }, nullptr);
-	k_timer_start(&sMeasurementsTimer, K_MSEC(kMeasurementsIntervalMs), K_MSEC(kMeasurementsIntervalMs));
+		[](k_timer *) { sAppTask.PostEvent(AppEvent{ AppEvent::MeasurementsTimer }); },
+		nullptr);
+	k_timer_start(&sMeasurementsTimer, K_MSEC(kMeasurementsIntervalMs),
+		      K_MSEC(kMeasurementsIntervalMs));
 
 	/* Init ZCL Data Model and start server */
 	chip::Server::GetInstance().Init();
 
 	/* Initialize device attestation config */
 	SetDeviceAttestationCredentialsProvider(Examples::GetExampleDACProvider());
 	ConfigurationMgr().LogDeviceConfig();
-	PrintOnboardingCodes(chip::RendezvousInformationFlags(chip::RendezvousInformationFlag::kBLE));
+	PrintOnboardingCodes(
```

Comment:
Was it formatted using `.clang-format` file from `samples/matter`? I'm quite surprised by the number of formatting changes :)

Reply:
I think it wasn't as it is located in applications directory, so probably other .clang-format rules were used